### ISSUE TITLE
VoiceStateUpdate Event Possibly Broken?

### ISSUE THREAD

**Issue by Lukeos11** [discordjs/discord.js#5378]
**Please describe the problem you are having in as much detail as possible:**

Hello All!
When join and leave a voice chat my bot returns undefined for newState and oldState.
I don't know if this is a problem with discord.js or my code.
Sorry, this is my first time creating an issue on Github.
Thanks, Luke.

**Include a reproducible code sample here, if possible:**

```js
client.on('voiceStateUpdate', (oldstate, newstate) => {
    console.log(oldstate);
    console.log(newstate);
    if(oldstate.voiceChannel === undefined && newstate.voiceChannel !== undefined) {
    // other code
    } else if (newstate.voiceChannel === undefined) {
    // other code
    }
});
```

**Further details:**

- discord.js version: V12.5.1
- Node.js version: V14.15.1
- Operating system: Windows 10
- Priority this issue should have – please be realistic and elaborate if possible: Low - Medium.

**Relevant client options:**

- partials: none
- gateway intents: none
- other: none

**Comment by extroonie** [comment]
No, it's not a bug. If you actually look into the docs of [VoiceState](https://discord.js.org/#/docs/main/stable/class/VoiceState), there no `voiceChannel` property. If you're looking for channel of the respective voice state - only `channel` and `channelID` are relevant.


If you're unaware about the changes made to v11 -> v12, please check out the guide on [Updating from v11 to v12](https://discordjs.guide/additional-info/changes-in-v12.html).
